driver.go

```diff
@@ -36,7 +35,7 @@ func (d ipfsDriver) Create(r volume.Request) volume.Response {
 		return volume.Response{}
 	}
 
-	volumePath := filepath.Join(d.mountPoint, volumeName)
+	volumePath := volumeName
```

Comment:
Because you can then have the possibility to use [IPNS](https://github.com/ipfs/examples/tree/master/examples/ipns).